### DISCUSSION TITLE
Load additional vcfs from AnVIL to seqr

### DISCUSSION THREAD

**Discussion by zihhuafang** [broadinstitute/seqr#2261] [Q&A] [answered]
Hi,

I have loaded one joint-called vcf from my AnVIL/Terra workspace to seqr. I would like to replace the one that has been loaded to seqr with the new joint-called vcf because we have more families sequenced in the same project. However, I cannot find a way to include new vcf, which I put in my workspace bucket, in seqr. With "edit" individuals, I can only add metadata for individuals but not to add new vcf. I also cannot create a new project with the same AnVIL workspace. The number of families sequenced will also grow on a regular basis, so we expect to add new vcf and individuals.

Could not find documentation how it works. Could anyone who has the similar experience advice me what to do?
Thanks in advance.
Best wishes,
Zih-Hua

**Comment by hanars** [comment, marked as answer]
Hi Zih-Hua,

Loading multiple VCFs to a seqr project is currently not supported. Given that it looks like you only have one sample in the existing project and have not really started analysis on it, I would recommend you put your new joint called VCF in a new AnVIL workspace and go through the loading process there, which will create a new project with all your samples. If you really want to use the existing AnVIL workspace (which I discourage, as having real data in projects named ""test" is a bad idea in general) I can delete the existing seqr project for you which will enable you to create a new project from the same workspace with a different set of individuals.

With regards to "The number of families sequenced will also grow on a regular basis, so we expect to add new vcf and individuals", we may need to discuss some alternative approaches to support this workflow for you. We hope sometime in 2022 to add better support for adding additional VCFs to existing seqr projects, but cannot commit to a date when this support will be available. We also ask that you keep in mind that while we do not charge to use seqr, loading data is not free for us. Frequently adding small amounts of data is not something we currently have the capacity to support. If you would like to reach out to our team to discuss a partnership in which we can regularly support adding incremental data, I can put you in touch with the appropriate people. If you would like to continue using seqr freely as it is, we recommend that as new families are sequenced you wait until you have a critical mass, joint call those families together into a new workspace, and then load those families into a new seqr project. Note that there is functionality in seqr to support searching and viewing data across multiple projects. We ask that you do this no more frequently than monthly. 

We are so excited that you have chosen to use seqr, and recognize that these limitations are disappointing. If the solutions I proposed here do not work well for you and you would like to talk in more detail about your needs, I am happy to meet with you to discuss.

Best,
Hana Snow
Technical Lead, seqr

**Reply by zihhuafang** [reply]
Dear Hana,

Thank you very much for the detailed explanation. I fully understand the current limitations and appreciate the supports. As I just wanted to explore how to employ seqr with my current AnVIL workspace, I chose to do a test without knowing the consequences not be able to modify the project with different sets of vcfs in the same workspace. If you could kindly delete my existing seqr project so I could start a new project with the real data, that would be really great. 

I will discuss with my team about the needs of our project and will reach out once we decide on the specifics.
Should I send an email to this email address: seqr@broadinstitute.org? 

Thank you again.
Zih-Hua

**Reply by hanars** [reply]
Hi Zih-Hua,

I have deleted your test project so you should be all set to re-load the project with your real data. 
And yes, that email address is a good place to reach out once you have specific requests for our team.
